catalogue/webapp/services/prismic/transformers/index.ts

```diff
@@ -39,7 +39,7 @@ export async function transformPrismicResponse(
       firstPublicationDate,
       contributors: allContributors,
       type,
-      summary: image?.caption[0].text,
+      summary: image?.caption ? image.caption[0].text : undefined,
```

Comment:
would this do the same thing without the need for the ternary?

`image?.caption?.[0]?.text`

Reply:
It would and I went with that at first and then changed it cause I thought maybe it was more readable at first glance?
But maybe not. So many question marks.

Reply:
my inclination is that once you're optional chaining you should just roll with it for as long as you can!